src/App.tsx

```diff
@@ -1,25 +1,107 @@
+import { FormEvent, useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+export const App: React.FC = () => {
+  const [titleText, setTitleText] = useState('');
+  const [selectedUser, setSelectedUser] = useState('');
```

Comment:
```suggestion
  const [selectedUser, setSelectedUser] = useState<User>(null);
```
you should write type for the user